packages/@o3r/components/package.json

```diff
@@ -37,6 +37,7 @@
     "@o3r/testing": "workspace:^",
     "@schematics/angular": "~16.0.5",
     "chokidar": "^3.5.2",
+    "comment-json": "^4.1.0",
```

Comment:
Should be by this package `ng-add` or one of it's dep's `ng-add` otherwise it will be complicated for the user :S
(it is in the `@o3r/core` dep but it will not be enough for `pnp` :S)

Reply:
it is in `@o3r/schematics` `ng-add`